lint.go

```diff
@@ -425,6 +425,11 @@ func (f *file) lintPackageComment() {
 	// Only non-main packages need to keep to this form.
 	if !f.pkg.main && !strings.HasPrefix(s, prefix) {
 		f.errorf(f.f.Doc, 1, link(ref), category("comments"), `package comment should be of the form "%s..."`, prefix)
+		return
+	}
+
+	if !f.pkg.main && strings.HasPrefix(s, prefix+"...") {
+		f.errorf(f.f.Doc, 1, link(ref), category("comments"), "package comment should be complete sentences")
```

Comment:
"should be a complete sentence"

Reply:
Fixed.

Reply:
Forgot to update the tests. Will fix this in a few hours.

Reply:
Done.